doc/modules/language-guide/pages/compiler-ref.adoc

```diff
@@ -94,6 +96,9 @@ You can use the following options with the `+moc+` command.
 
 |`+--release+` |Ignores debug expressions in the source.
 
+|`+--sanity-checks+` |Enable sanity checking in the runtime system and generated code (for compiler development only).
+
+
```

Comment:
Should we document dev-only features in language guide? Do we do this for any other flag currently?

Reply:
Yeah, we seems to document all them (though I really think we shouldn't and maybe  hide some of them from --help). I think this ones particularly bad in case someone uses it (for reassurance) and complains about the cost...

Reply:
Done